microsetta_admin/templates/scan.html

```diff
@@ -2,5 +2,90 @@
 {% block content %}
 <h3>Microsetta Scan</h3>
 <div>
+    <form name="search_form" id="search_form" method="GET">
+        <table>
+            <tr>
+                <td><label for="sample_barcode">Barcode: </label></td>
+                <td><input type="text" name="sample_barcode" id="sample_barcode" {% if info %} value="{{info[0].barcode}}"{% endif %}></td>
+            </tr>
+            <tr>
+                <td></td>
+                <td><input type="submit" value="Find Sample"></td>
+            </tr>
+        </table>
+        {% if search_error %}
+            <p style="color:red">
+            {{search_error |e}}
+            </p>
+        {% endif %}
+        <script>
+           document.agForm.sample_barcode.focus()
+        </script>
+    </form>
+
+    <br>
+    <hr>
+    <br>
+
+    {% if info %}
+        <form action="/scan" name="update_form" id="update_form" method="post">
+        <input type="hidden" name="sample_barcode" value="{{info[0].barcode}}"/>
+        <table>
+            <tr>
+                <td>Barcode: </td>
+                <td>{{info[0].barcode}}</td>
+            </tr>
+            <tr>
+                <td>Project: </td>
+                <td>{{info[0].project}}</td>
```

Comment:
how are multiple projects shown?

Reply:
Ugh.  Yeah the model passed down over the api for this wasn't great.  I've set it to return all the projects within a single barcode_info, which will remove all the info[0] silliness, but this PR is now dependent on a PR going into microsetta-private-api first.

Reply:
Merged it